### ISSUE TITLE
Discriminator weights for pretrained models

### ISSUE THREAD

**Issue by ctlaltdefeat** [jik876/hifi-gan#5]
It'd be nice if you uploaded the discriminator model weights for the pretrained models, so that users can more easily fine-tune the models for their datasets.

**Comment by jik876** [comment]
We're re-training the universal model, and we'll share it once it's completed successfully.

**Comment by jik876** [comment]
I've updated the pretrained weights files of the universal model. It will be a good basis for transfer learning to other datasets.
It may need to adjust the initial epoch or the initial learning rate to take into account the learning rate scheduling, when it used for transfer learning.
